spec/artsy-eventstream/artsy/publisher_spec.rb

```diff
@@ -29,7 +29,7 @@
       allow(event).to receive(:verb).and_return(nil)
       expect { Artsy::EventService::Publisher.publish(topic: 'test', event: event) }.to raise_error 'Event missing topic or verb.'
     end
-    it 'calls publish on the exchange with proper data' do
+    it 'uses verb as routing key when calling publish on the exchange without passing routing_key' do
```

Comment:
@joeyAghion so what's the verdict, if we're keeping `verb` as an option to set the `routing_key`. any objections to merging this?